src/shared/configs/database.config.ts

```diff
@@ -0,0 +1,16 @@
+import { ConfigService } from '@nestjs/config';
+import { DataSourceOptions } from 'typeorm';
+
+export const databaseConfig = (
+  configService: ConfigService,
+): DataSourceOptions => ({
+  type: 'postgres',
+  host: configService.getOrThrow('DB_HOST'),
+  port: configService.getOrThrow('DB_PORT'),
+  username: configService.getOrThrow('DB_USER'),
```

Comment:
Don't use getOrThrow, just use get. You are validating the config during the app start